MasterPassword/C/mpw-cli.c

```diff
@@ -96,7 +96,7 @@ static char *getline_prompt(const char *prompt) {
 int main(int argc, char *const argv[]) {
 
     // Read the environment.
-    char *fullName = strdup( getenv( MP_env_fullname ) );
+    char *fullName = NULL;
```

Comment:
It should be null to prevent another leak memory:

```
 switch (opt) {
            case 'u':
                fullName = strdup( optarg );
                break;
```